src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -602,15 +551,15 @@ export class StudyViewPageStore {
 
     @autobind
     public async openComparisonPage(params:{
-        type:ChartTypeEnum.PIE_CHART|ChartTypeEnum.TABLE|ChartTypeEnum.BAR_CHART,
-        clinicalAttribute: ClinicalAttribute,
+        chartMeta: ChartMeta,
+        clinicalAttribute?: ClinicalAttribute,
```

Comment:
I think there is no need to pass `clinicalAttribute` as `chartMeta` would be having it.